.github/workflows/mirror.yml

```diff
@@ -0,0 +1,41 @@
+name: Mirror
+
+permissions:
+  contents: write
+  pull-requests: read
+  actions: write
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  push:
+    branches:
+      - "release/*"
+    tags:
+      - "v*.*.*"
+  workflow_dispatch: {}
+
+jobs:
+  mirror:
+    timeout-minutes: 5
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          persist-credentials: false
+      - name: Mirror
+        uses: addnab/docker-run-action@v3
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: gcr.io
+          image: cnrancher/hangar:v1.6.0
+          options: -v ${{ github.workspace }}:/images -e DEST_REGISTRY=docker.io -e DEST_USERNAME=${{ secrets.CI_DOCKERHUB_USERNAME }} -e DEST_PASSWORD=${{ secrets.CI_DOCKERHUB_PASSWORD }}
+          run: |
+            hangar mirror -f /images/hack/mirror/mirror_image_list.txt
```

Comment:
do we try `crane copy` before? it is a docker-less tool.
https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_copy.md

Reply:
hangar can do multi-arch image copy with a simple `image_list` file without caring about manifest file.

Reply:
try our best to match the following content.
1. docker-less manifest images transferring.
2. keep all source/destination images in the form of `registry.io/namespace/sub-namespace/repo:tag`.
<img width="829" alt="image" src="https://github.com/seal-io/walrus/assets/3394724/fb14d824-bb06-47f7-89b3-0209ed1fbb80">
3. allow mutating the image's tag of src/dest.